app/assets/javascripts/spree/backend/spree_volume_pricing.js

```diff
@@ -14,3 +14,76 @@ $(function () {
   });
 
 });
+
+$(document).ready(function () {
+  'use strict';
+
+  function formatUser(user) {
+    return Select2.util.escapeMarkup(user.email);
+  }
+
+  if ($('#pricing_tier_user_ids').length > 0) {
+    $('#pricing_tier_user_ids').select2({
+      placeholder: 'Choose an user',
+      minimumInputLength: 1,
+      multiple: true,
+      initSelection: function (element, callback) {
+        var url = Spree.url(Spree.routes.users_api, {
+          ids: element.val(),
+          token: Spree.api_key
+        });
+        return $.getJSON(url, null, function (data) {
+          return callback(data.users);
+        });
+      },
+      ajax: {
+        url: Spree.routes.users_api,
+        quietMillis: 200,
+        datatype: 'json',
+        data: function (term) {
+          return {
+            q: {
+              email_cont: term
+            },
+            token: Spree.api_key
+          };
+        },
+        results: function (data) {
+          return {
+            results: data.users
+          };
+        }
+      },
+      formatResult: formatUser,
+      formatSelection: formatUser
+    });
+  }
+
+  $(document).on('change', '.supplier_select', function(e) {
```

Comment:
make sure we this works with turbolinks

Reply:
@brettallred spree don't use turbolinks in backend  layout. Spree also using these everywhere in backend js files